testing-modules/junit-5-basics-2/src/test/java/com/baeldung/junit/norunnablemethods/JUnit5UnitTest.java

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.junit.norunnablemethods;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class JUnit5UnitTest {
+
+    private static int expected = 1;
+
+    @BeforeAll
+    public static void initialize() {
+        System.out.println("Intializes test setup");
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        System.out.println("Test clean-up");
+    }
+
+    @BeforeEach
+    public void initTest() {
+        expected = 2;
+    }
+
+    @AfterEach
+    public void cleanUp() {
+        expected = 0;
```

Comment:
What is the purpose of this?

Reply:
Removed this class as it is redundant. Previously, I added this class to refer only to the import statements.